internal/connector/noop/noop_connector.go

```diff
@@ -1,6 +1,12 @@
 package noop
 
-import "github.com/pingidentity/pingctl/internal/connector"
```

Comment:
If you want to delete this file here or in a future MR feel free.

Reply:
Deleted!